### ISSUE TITLE
[Accessibility: ACC-264.1] Button: UI name not read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12575]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Name of the UI should be read out to the user

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All versions.

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [VHD](https://sap.github.io/fundamental-ngx/#/platform/value-help-dialog)
Access the component using screen reader, UI name is not read for ‘Clear/Remove’ button associated to ‘Selected Items and Conditions’ tokenizer text box.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/c7fb156a-dd09-456b-bb57-bcf67c8c0a98">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 The button is announced: Button clear selected items. What else do you expect here?
@nikolay-kolarov do you think we are missing something?

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

The button is unlabelled and what you see is a description that is announced from the tooltip, after the button role.
In such cases, we set the tooltip also as aria-label, as it will label the button as well. And then as it is the same, the screen reader ignores the tooltip and does not produce duplicated announcement.
You can see this in the Remove All label/tooltip here:
https://ui5.sap.com/#/entity/sap.ui.comp.valuehelpdialog.ValueHelpDialog/sample/sap.ui.comp.sample.valuehelpdialog.recommended
![image](https://github.com/user-attachments/assets/330c9c5f-7eb1-4d15-8945-13e49deb606a)
And on your side the aria-label is empty:
![image](https://github.com/user-attachments/assets/21b4ce33-031c-4dff-ac6c-acf8fec2aff8)

Best Regards,
Nikolay
